docs/content/theory.md

```diff
@@ -0,0 +1,189 @@
+# Theory
```

Comment:
Markdown is highly preferred! Otherwise, you won't be able to take advantage of all the [MyST parser](https://myst-parser.readthedocs.io/en/latest/) functionality.